tests/test_schema_validation.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gordon.schema.validate import JsonValidator
+from gordon import exceptions
+
+import pytest
+
+
+@pytest.fixture
```

Comment:
I think you can scope this to session, i.e. `@pytest.fixture(scope='session')`, so the fixture is only called once during the entire run of tests, rather than each time. Unless there's state that might pollute it.

Reply:
Sounds good. There's no change of state, and it could make the tests run a bit faster.